.github/workflows/integration-test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   test:
     name: "Integration Test"
     runs-on: ${{ matrix.operating-system }}
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
it didn't got slower with this change. [recent PRs](https://github.com/ondrejmirtes/simple-downgrader/pull/11) had a similar timeout